src/stir_ldt/mod.rs

```diff
@@ -16,6 +17,7 @@ where
     MerkleConfig: Config<Leaf = [F]>,
 {
     merkle_proofs: Vec<(MultiPath<MerkleConfig>, Vec<Vec<F>>)>,
+    first_round_coeffs: Option<Vec<DensePolynomial<F>>>,
```

Comment:
Why does this need a addition to the proof? In the WHIR impl we simply do it within the (already existing) Merkle paths, we shouldn't need it.